packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -75,6 +88,114 @@ ModifierButton.propTypes = {
   }).isRequired,
 };
 
+const isBlockActive = (editor, type, level = undefined) => {
```

Comment:
The `level` param only makes sense for the heading type. Likewise, it's possible that several other types that will need to check other attributes to check if the block is active (like list checking if it's ordered/unordered). So I don't think we can keep adding params to that function.

I would recommend that instead of `type, level`, you put a single `value` param, and here when you call this function it would be `blockItems[value].value`. Value would be an object with type but also attributes specific to that type, that you can use inside the cases to make the match condition